### DISCUSSION TITLE
Welcome to ML-Gallery Discussions!

### DISCUSSION THREAD

**Discussion by BlueBlaze6335** [gdscjgec/ML-Gallery#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Join the community - [GDSC JGEC](https://gdsc.community.dev/jalpaiguri-government-engineering-college-jalpaiguri/)
Join our Discord - [Invite Link](https://discord.gg/kTHMgAgmvf)
Ask questions here - [Link to Discord text channel](https://discord.gg/ZZpUQFXUBZ)

**Comment by Kasrazeba-Mulla** [comment]
👋 Hi, I’m @Kasrazeba-Mulla
👀 I’m interested in JavaScript,Python ,HTML,CSS
🌱 I’m currently learning Web Development and Machine Learning

**Comment by Tanmoydey21** [comment]
- 👋 Hi, I’m @Tanmoydey21
- 👀 I’m interested in Python, C++, Internet of things, Machine Learning
- 🌱 I’m currently learning Internet of things
- 💞️ I’m looking to collaborate on ...
- 📫 How to reach me , my email is deytanmoy2002@gmail.com
